### PR TITLE
build: create data package

### PR DESCRIPTION
**Pull request by balajtimate**
### Description

- Add `__init__.py` to `data` directory, to mark is as subpackage and include it in conda builds
- Update versioning in `htsinfer/__init__.py`

### Type of change

- [X] New feature (non-breaking change which adds functionality)

### Checklist

Please carefully read these items and tick them off if the statements are true
_or do not apply_.

- [X] I have performed a self-review of my own code
- [X] My code follows the existing coding style, lints and generates no new
      warnings
- [X] I have added type annotations to all function/method signatures, and I
      have added type annotations for any local variables that are non-trivial,
      potentially ambiguous or might otherwise benefit from explicit typing.
- [X] I have commented my code in hard-to-understand areas
- [X] I have added ["Google-style docstrings"] to all new modules, classes,
      methods/functions or updated previously existing ones
- [X] I have added tests that prove my fix is effective or that my feature
      works
- [X] New and existing unit tests pass locally with my changes and I have not
      reduced the code coverage relative to the previous state
- [X] I have updated any sections of the app's documentation that are affected
      by the proposed changes

If for some reason you are unable to tick off all boxes, please leave a
comment explaining the issue you are facing so that we can work on it
together.

### CODE REVIEW

**Comment by balajtimate** [comment]
There's a temporary issue with coverage, but the unit tests run normally, the report just cannot be submitted.

**Comment by uniqueg** [comment]
Yeah, I had the same. Annoying 😅

File: htsinfer/__init__.py [hunk 1]
```diff
@@ -2,4 +2,4 @@
 
 from htsinfer.htsinfer import HtsInfer  # noqa:F401
 
-__version__ = "0.9.0"
+__version__ = "0.10.0"
```
Comment (uniqueg) [inline htsinfer/__init__.py:5]: Why is this from 9 to 10? I thought we were already at 10? :)

Comment (balajtimate) [reply htsinfer/__init__.py:5]: Yes, but it wasn't updated here, we would re-release `v0.10.0`

**Review by uniqueg — APPROVED** [review]

**Pull request merged**